recirq/otoc/loschmidt/tilted_square_lattice/analysis.py

```diff
@@ -0,0 +1,172 @@
+from typing import Callable, Dict, cast, Sequence, Any, Tuple, List
+
+import numpy as np
+import pandas as pd
+from scipy.optimize import curve_fit
+
+import cirq
+from cirq_google.workflow import ExecutableGroupResult, SharedRuntimeInfo, \
+    QuantumRuntimeConfiguration, \
+    ExecutableResult
+from recirq.otoc.loschmidt.tilted_square_lattice import TiltedSquareLatticeLoschmidtSpec
+
+
+def to_ground_state_prob(result: cirq.Result) -> float:
+    """The loschmidt echo success probability is the fraction of times we return to the state
```

Comment:
If possible, keep summary to one line.

Reply:
reduced summary line, moved some of the explanation to the next paragraph